src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * @author Ririan
+ */
+import React from 'react'
+import {Trans} from '@lingui/react'
+import {t} from '@lingui/macro'
+import {List} from 'semantic-ui-react'
+import Module from 'parser/core/Module'
+import {getDataBy} from 'data'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import STATUSES from 'data/STATUSES'
+import ACTIONS from 'data/ACTIONS'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+
+// All of Bards DoTs
+const DOTS = [
+	STATUSES.CAUSTIC_BITE.id,
+	STATUSES.STORMBITE.id,
+	STATUSES.VENOMOUS_BITE.id,
+	STATUSES.WINDBITE.id,
+]
+
+export default class Sidewinder extends Module {
+	static handle = 'sidewinder'
+	static title = t('brd.sidewinder.title')`Sidewinders and Shadowbites`
+	static dependencies = [
+		'suggestions',
+	]
+
+	_amountOfBadSidewinders = 0
+	_amountOfBadShadowbites = 0
+
+	_badCasts = []
+
+	_enemies = {}
+
+	constructor(...args) {
+		super(...args)
+
+		this.addHook('cast', {
+			by: 'player',
+			abilityId: [ACTIONS.SIDEWINDER.id, ACTIONS.SHADOWBITE.id],
+		}, this._onSidewinderCast)
+
+		this.addHook('applydebuff', {
+			by: 'player',
+			abilityId: DOTS,
+		}, this._onApplyDot)
+
+		this.addHook('removedebuff', {
+			by: 'player',
+			abilityId: DOTS,
+		}, this._onRemoveDot)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onSidewinderCast(event) {
+		const enemyDots = this._getEnemyDots(event.targetID)
+		const abilityId = event.ability.guid
+		if (enemyDots.length < 2) {
+			if (abilityId === ACTIONS.SIDEWINDER.id) {
+				this._amountOfBadSidewinders++
+			} else {
+				this._amountOfBadShadowbites++
+			}
+
+			this._badCasts.push({abilityId: abilityId, dotsApplied: enemyDots.length, appliedDot: enemyDots[0], timestamp: event.timestamp})
+		}
+	}
+
+	_onApplyDot(event) {
+		const enemyDots = this._getEnemyDots(event.targetID)
+		enemyDots.push(event.ability.guid)
+	}
+
+	_onRemoveDot(event) {
+		const enemyDots = this._getEnemyDots(event.targetID)
+		const dotIndex = this._findDot(enemyDots)
+		if (dotIndex !== -1) {
+			enemyDots.splice(dotIndex, 1)
+		}
+	}
+
+	// Returns the enemy statuses state
+	_getEnemyDots(targetId) {
+		if (!this._enemies[targetId]) {
+			this._enemies[targetId] = []
+		}
+
+		return this._enemies[targetId]
+	}
+
+	_findDot(array) {
+		return array.findIndex(element => {
+			for (const dot of DOTS) {
+				if (element === dot) {
+					return true
+				}
+			}
+			return false
+		})
+	}
+
+	_onComplete() {
+		if (!this._badCasts.length) {
+			return
+		}
+		if (this._amountOfBadSidewinders) {
```

Comment:
Don't need to wrap `TieredSuggestion` in a conditional, it'll automatically hide if it doesn't meet the lowest tier.